cafebabel/articles/tags/views.py

```diff
@@ -36,3 +37,16 @@ def detail(slug):
     articles = Article.objects(tags__in=[tag], status='published')
     return render_template('articles/tags/detail.html', tag=tag,
                            articles=articles)
+
+
+@tags.route('/<slug>/edit/', methods=['get', 'post'])
+@editor_required
+def edit(slug):
+    tag = Tag.objects.get_or_404(slug=slug)
```

Comment:
Won't it break if we have the same slug in 2 languages? ie: _meetmyhood_ that will remain the same throughout languages

Reply:
domain name (.fr, .en, .de..) will do so it's not the same slug, won't it ?

Reply:
Indeed, to select the slug based on the domain we need to filter it out. That's what I [suggested in this commit](https://github.com/cafebabel/cafebabel.com/pull/153/files#diff-a0088a31e32d86af81efe546d99ad4abR45).